src/assets/index.ts

```diff
@@ -1,7 +1,7 @@
-export const Logo = `./icons/logo.svg`;
-export const Minus = `./icons/minus.svg`;
-export const Plus = `./icons/plus.svg`;
-export const Info = `./icons/info.svg`;
-export const Back = `./icons/back.svg`;
+export const Logo = import.meta.env.BASE_URL + `icons/logo.svg`;
+export const Minus = import.meta.env.BASE_URL + `icons/minus.svg`;
+export const Plus = import.meta.env.BASE_URL + `icons/plus.svg`;
+export const Info = import.meta.env.BASE_URL + `icons/info.svg`;
+export const Back = import.meta.env.BASE_URL + `icons/back.svg`;
 
-export const Default = `./images/default.png`;
+export const Default = import.meta.env.BASE_URL + `images/default.png`;
```

Comment:
BASE_URL을 하나의 변수로 만들어서 관리해주면 어떨까요?

```suggestion
const BASE_URL = import.meta.env.BASE_URL ?? '/';
export const Logo = `${BASE_URL}icons/logo.svg`;
export const Minus = `${BASE_URL}icons/minus.svg`;
export const Plus = `${BASE_URL}icons/plus.svg`;
export const Info = `${BASE_URL}icons/info.svg`;
export const Back = `${BASE_URL}icons/back.svg`;
export const Default = `${BASE_URL}images/default.png`;
```

Reply:
BASE_URL 상수로 분리하였습니다!